main.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'app'
+
+def ask_user
+  puts 'Select what you want to do'
+  puts "
+    1 List all books.
+    2 List all people.
+    3 Create a person.
+    4 Create a book.
+    5 Create a rental.
+    6 List all rentals for a given person id.
+    7 Exit."
+  gets.chomp.to_i
+end
+
+def loop_books(my_app)
+  my_app.books.each_with_index do |book, index|
+    puts "#{index + 1}) Title: #{book.title}, Author: #{book.author}"
+  end
+end
+
+def loop_people(my_app)
+  my_app.people.each_with_index do |person, index|
+    type = person.is_a?(Student) ? 'Student' : 'Teacher'
+    puts "#{index}) [#{type}] Name: #{person.name} ID: #{person.id} Age: #{person.age}"
+  end
+end
+
+def process_choice(choice, my_app)
+  case choice
+  when 1
+    loop_books(my_app)
+  when 2
+    loop_people(my_app)
+  when 3
+    my_app.create_person
+  when 4
+    my_app.create_book
+  when 5
+    my_app.create_rental
+  when 6
+    puts my_app.view_rental
+  end
+end
+
```

Comment:
- [ ] When I run your app and try to list the book by selecting the first option the app crashes and exits. Kindly fix that.

  ![image](https://github.com/citec-47/school-library/assets/54848363/b1330666-4c70-43bc-a79e-da8d2122491c)

- [ ] The issue above is also there for the list people option, 2. Kindly fix that.

- [ ] I am also unable to create a rental after creating books and people. The app crashes. Kindly fix that.

- [ ] Since I could not create a rental I also could check the list rental option. Kindly revisit [the video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) shared with you and ensure your app works are it is presented in that video.